mnemosyne/core/cost_log.py

```
@@ -3,11 +3,10 @@
 Tracks memory injection costs over time for benchmarking.
 """
 
-import json
 import sqlite3
 from datetime import datetime
 from pathlib import Path
-from typing import Dict, List
+from typing import Dict
```

Ruff (0.15.21)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)